src/diagnostics/land_compute_methods.jl

```diff
@@ -78,7 +78,20 @@ end
 
 # Canopy - Conductance
 @diagnostic_compute "stomatal_conductance" Union{SoilCanopyModel, LandModel} p.canopy.conductance.gs
-@diagnostic_compute "canopy_transpiration" Union{SoilCanopyModel, LandModel} p.canopy.energy.turbulent_fluxes.transpiration
+
+function compute_canopy_transpiration!(
+    out,
+    Y,
+    p,
+    t,
+    land_model::Union{SoilCanopyModel{FT}, LandModel{FT}},
+) where {FT}
+    if isnothing(out)
+        return p.canopy.energy.turbulent_fluxes.transpiration .* 1000
```

Comment:
could you add a comment explaining why `.* 1000`?

Reply:
added!